### ISSUE TITLE
Add systemd service

### ISSUE THREAD

**Issue by dos1** [joshkunz/ashuffle#73]
In order to have ashuffle ran at boot, I've configured following systemd service in `/etc/xdg/systemd/user/ashuffle.service` on my PC:

```
[Unit]
Description=Shuffle service for mpd
Requires=mpd.service
After=mpd.service

[Service]
ExecStart=/usr/bin/ashuffle

[Install]
WantedBy=default.target
```

Once #72 is resolved it may make sense to distribute such unit file together with ashuffle.

**Comment by joshkunz** [comment]
`ashuffle` doesn't really have a "distribution" per-se, so I'm not sure where we could include it. One thing many projects do is maintain a "contrib" directory where community members can add scripts/extensions/documentation that are not part of the core project, but may be helpful for others. I think this is a good candidate for something like that.

Alternatively, this could go in say, a wiki, with other related installation instructions. I'm not sure which yet... I'll think about it.

**Comment by joshkunz** [comment]
So thinking about it more, I like the idea of a `contrib` directory for this kind-of stuff. We could drop this in a `contrib/systemd/` directory along with a `readme.md` explaining how to install/use it. @dos1 would you be willing to write the `.service` file and open a PR? Otherwise I'll get to it eventually...

**Comment by joshkunz** [comment]
This is fixed as of PR #75. Thanks @j0hnnybash!
